### PR TITLE
enh(zendesk): lower concurrency to 3 for fetching ticket comments

### PR DESCRIPTION
**Pull request by fontanierh**
## Description

We're seeing EPIPE errors and timeouts.

I suspect it is related to the 10 concurrent requests to zendesk API for comments

## Risk

N/A

## Deploy Plan

Deploy connectors

### CODE REVIEW

File: connectors/src/connectors/zendesk/temporal/activities.ts [hunk 1]
```diff
@@ -434,10 +435,20 @@ export async function syncZendeskTicketBatchActivity({
...
-    async (ticket) => {
-      const comments = await zendeskApiClient.tickets.getComments(ticket.id);
+    async (ticket) => zendeskApiClient.tickets.getComments(ticket.id),
+    { concurrency: 3, onBatchComplete: heartbeat }
+  );
+
+  const res = await concurrentExecutor(
+    _.zip(tickets, comments),
+    async ([ticket, comments]) => {
+      if (!ticket || !comments) {
```
Comment (fontanierh) [inline connectors/src/connectors/zendesk/temporal/activities.ts:447]: Both are technically `| undefined` as there's no typescript guarantee that both arrays are equal size

**Review by aubin-tchoi — APPROVED** [review]
LGTM

File: connectors/src/connectors/zendesk/temporal/activities.ts (hunk 1 above)
Comment (aubin-tchoi) [reply connectors/src/connectors/zendesk/temporal/activities.ts:447]: OOC: do you prefer this over a map over the tickets to turn them into objects?

Comment (fontanierh) [reply connectors/src/connectors/zendesk/temporal/activities.ts:447]: Shorter like this but I don't have a strong opinion really

Comment (aubin-tchoi) [reply connectors/src/connectors/zendesk/temporal/activities.ts:447]: fair 👍

**Pull request merged**